samples/instrumentation-quickstart/otel-collector-config.yaml

```diff
@@ -98,14 +98,14 @@ processors:
     send_batch_max_size: 500
     send_batch_size: 500
     timeout: 1s
-  # Make sure Google Managed service for Prometheus required labels are set
+  # Provide defaults for Google Managed Service for Prometheus labels
   resource:
     attributes:
-      - { key: "location", value: "us-central1", action: "upsert" }
-      - { key: "cluster", value: "no-cluster", action: "upsert" }
-      - { key: "namespace", value: "no-namespace", action: "upsert" }
-      - { key: "job", value: "us-job", action: "upsert" }
-      - { key: "instance", value: "us-instance", action: "upsert" }
+      - { key: "cloud.region", value: "us-central1", action: "insert" }
+      - { key: "k8s.cluster.name", value: "no-cluster", action: "insert" }
+      - { key: "k8s.namespace.name", value: "no-namespace", action: "insert" }
```

Comment:
Are the `k8s.*` keys preferred over the ones documented here https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-otel#detect-resource-attributes?

Reply:
I wanted these to be defaults, and not to override what users set/detect.  If we use the ones from that document, it will override anything from the user (e.g. `job` takes precedence over `service.name`).

Reply:
Are users expected to set `job` in their OTel resource?

Reply:
They are not expected to set job.